docs/guides/defaults-and-configuration-options.ipynb

```diff
@@ -186,7 +182,7 @@
     "    **Translation stage**\n",
     "  </summary>\n",
     "\n",
-    "- `translation_method` (str) - Name of translation pass (`basis_translator`, `translator`, or `synthesis`) This can also be the external plugin name to use for the translation stage. You can see a list of installed plugins by running `list_stage_plugins()` with `translation` for the `stage_name` argument. The default value is `basis_translator`.\n",
+    "- `translation_method` (str) - Name of translation pass (`translator`, or `synthesis`) This can also be the external plugin name to use for the translation stage. You can see a list of installed plugins by running `list_stage_plugins()` with `translation` for the `stage_name` argument. The default value is `translator`.\n",
```

Comment:
```suggestion
    "- `translation_method` (str) - Name of translation pass (`translator` or `synthesis`) This can also be the external plugin name to use for the translation stage. You can see a list of installed plugins by running `list_stage_plugins()` with `translation` for the `stage_name` argument. The default value is `translator`.\n",
```